### DISCUSSION TITLE
Help with auto complete and DDG Bangs

### DISCUSSION THREAD

**Discussion by czadikem** [searx/searx#2821] [Installation issue: Docker]
Hi there I have just got searx setup using docker like it says in the documentation and I notice that when I enable autocomplete it doesn't work and also DDG bangs don't seem to work.  How can I fix this?  Am I missing something?  Any help would be appreciated.

**Comment by allendema** [comment]
Have you tried other autocomplete engines?

SearX supports these [bangs](https://raw.githubusercontent.com/searx/searx/606aa79e4949cd29d207443a3f82c368fbae2faf/searx/data/bangs.json) which can be triggered with ```!!engine yourquery ```, example:

``` !!gnu free ```
